src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -3268,7 +3268,7 @@ export class ResultsViewPageStore {
         {}
     );
 
-    readonly studyToMolecularProfileDiscrete = remoteData<{
+    readonly studyToMolecularProfileDiscreteCna = remoteData<{
```

Comment:
Seems like here what we mean by discrete is actually discrete CNA. @adamabeshouse @alisman I couldn't find usage intending for another discrete data type, but just in case can you confirm that we use this only for discrete CNA data, not for any other discrete data?

Reply:
@onursumer yes 100%